internal/core/firefly_config.go

```diff
@@ -236,7 +237,8 @@ func NewFireflyConfig(stack *types.Stack, member *types.Member) *FireflyConfig {
 		memberConfig.Database = &DatabaseConfig{
 			Type: "postgres",
 			PostgreSQL: &CommonDBConfig{
-				URL: getPostgresURL(member),
+				URL:      getPostgresURL(member),
+				MaxConns: 50,
```

Comment:
Shouldn't this take the value from the config, but with a default of `50` set?

Reply:
ah, ok - one sec ... maybe this code uses config different. Let me take a look around the change.

Reply:
oh - sorry. Ok, completely my bad. I assumed I was looking at `firefly` (not `firefly-cli`).

I think this code change should be in FireFly (not the CLI).

Reply:
See comment here: https://github.com/hyperledger/firefly/blob/4f4390336ee0b6c9f68b5ed2b79ffc3fbbf293d1/internal/database/sqlcommon/config.go#L44

... I think we've learned PostgreSQL should be in the group that sets a default

Reply:
https://github.com/hyperledger/firefly/pull/385 Made a PR in firefly for this. Once merged I'll close this PR